swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -262,9 +262,9 @@ protected SSListItem createNullItem(Model.Remodel remodel) {
 	protected ArrayList<String> options = null;
 
 	/**
-	 * The column name whose value is written back to the database when the user
-	 * chooses an item in the combo box. This is generally the PK of the table to
-	 * which a foreign key is mapped.
+	 * The column name used to query the values for the bound column mappings.
```

Comment:
Documentation for this was very poor in my opinion. I momentarily thought it was duplicating boundColumn in SSCommon and should be removed.